docs/concepts/concept-pt2-range-encode-bit-slice.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Part 2 - bit-slicing and range-encoding
+layout: default
+parent: Bitmap index overview
+grand_parent: Concepts
+nav_order: 2
+---
+
```

Comment:
I'm going to admit it now: I suspect this breaks my own rule for endless scroll.
For the moment I want to leave as-is because it needs a review for truth and accuracy.
Post review I'll split it up probably into individual pages based on the "step" headings

Reply:
I think with the removal of range encoding this page is (a) better and (b) not so much of an endless scroll.
What do you think?